src/Myra/Graphics2D/UI/TextBlock.cs

```diff
@@ -87,6 +93,51 @@ public bool Wrap
 			}
 		}
 
+		/// <summary>
+		/// Replaces overflowing text with an ellipsis.
+		/// </summary>
+		[EditCategory("Appearance")]
+		[DefaultValue(false)]    
+		public bool AutoEllipsis
```

Comment:
I think it makes more sense to add third enum value(AutoEllipsisMethod.Note) rather than this property